specification/src/main/asciidoc/jakarta-concurrency.adoc

```diff
@@ -2452,16 +2265,16 @@ each task provider’s implementation.
 [arabic]
 . A task instance that starts a transaction must complete the
 transaction before starting a new transaction.
-. The task provider uses the javax.transaction.UserTransaction interface
+. The task provider uses the `jakarta.transaction.UserTransaction` interface
 to demarcate transactions.
-. Transactions are demarcated using the begin(), commit() , and
-rollback() methods of the UserTransaction interface.
+. Transactions are demarcated using the `begin()`, `commit()` , and
+`rollback()` methods of the UserTransaction interface.
 . While an instance is in an active transaction, resource-specific
 transaction demarcation APIs must not be used (e.g. if a
-javax.sql.Connection is enlisted in the transaction instance, the
-Connection.commit() and Connection.rollback() methods must not be used).
+`javax.sql.Connection` is enlisted in the transaction instance, the
```

Comment:
One more occurrence of the wrong package name (error is in the original spec, not introduced by these changes). Should be java.sql.Connection